src/classes/query-builder.js

```diff
@@ -1,89 +1,145 @@
+const _ = require('lodash')
```

Comment:
I think destructuring to a function in most cases is the way to go.
With lodash especially this means we could easily refactor to include only parts of lodash or to require modules of lodash via:
```js
const capitalize = require('lodash/capitalize')
```